pytest_fixtures/component/rh_cloud.py

```diff
@@ -45,7 +41,7 @@ def organization_ak_setup(rhcloud_sat_host, rhcloud_manifest_org):
     default_subscription = subscription.search(
         query={'search': f'name="{DEFAULT_SUBSCRIPTION_NAME}"'}
     )[0]
-    ak.add_subscriptions(data={'quantity': 10, 'subscription_id': default_subscription.id})
+    ak.add_subscriptions(data={'quantity': 2, 'subscription_id': default_subscription.id})
```

Comment:
I think if we use the SCA manifest, then we won't require to add subs like these, right?

Reply:
Yes, that's correct, but the scope of the manifester conversion effort was set at swapping out the manifests without modifying entitlement mode. To be clear, I understand that SCA mode will be mandatory at some point in the future, and I have raised this point myself several times. So while I agree with this change, I think it's outside the scope of this PR.